python/pyspark/tests/test_context.py

```diff
@@ -267,6 +267,13 @@ def test_resources(self):
             resources = sc.resources
             self.assertEqual(len(resources), 0)
 
+    def test_disallow_to_create_spark_context_in_executors(self):
+        with SparkContext("local-cluster[3, 1, 1024]") as sc:
```

Comment:
I would put a JIRA ID and a short comment like as we lately documented it at http://spark.apache.org/contributing.html

```python
def test_case(self):
    # SPARK-12345: a short description of the test
    ...
```